<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="m_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>391</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>991</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="m_chenxing">
     <attribute name="title">
      <string>晨星</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="m_tiantian">
     <attribute name="title">
      <string>天天</string>
     </attribute>
     <widget class="QLineEdit" name="m_tt_input_zhishu_name">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>指数名称</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_tt_btn_search_zhishu">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_tt_btn_search_detail">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>获取详情</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_tt_btn_search_drawdown">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>更新回撤</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="m_tt_funds">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>511</width>
        <height>301</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QLabel" name="m_tt_fund_info">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="m_tt_fund_cond">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="m_tt_fund_info_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>不包含</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="m_tt_ck_fenhong">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>分红</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="m_tt_ck_nianfei">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>管理费</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_tt_btn_fund_cond">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>应用条件</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="m_tt_ck_huiche">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>回撤</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_tt_btn_opendir">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>打开目录</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="m_tiantian_bond">
     <attribute name="title">
      <string>天天债券</string>
     </attribute>
     <widget class="QPushButton" name="m_tt_btn_search_bond2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>bond2</string>
      </property>
     </widget>
     <widget class="QComboBox" name="m_tt_bond_tp">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="m_tt_btn_search_bond">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>bond</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="m_zhongzheng">
     <attribute name="title">
      <string>中证指数</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="m_zhongzhai">
     <attribute name="title">
      <string>中债指数</string>
     </attribute>
     <widget class="QPushButton" name="m_info_bond_div_stock">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>显示债股比</string>
      </property>
     </widget>
     <widget class="QComboBox" name="m_comb_zz_show_list">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>321</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
